src/App.js

```diff
@@ -44,21 +89,37 @@ const Home = () => {
         <HomeCategoriesComponent categories={Categories.results} />
       </div>
       <div className="Products">
-        <FeaturedProductsComponent products={FeaturedProducts.results} />
+        <FeaturedProductsComponent products={FeaturedProductsArray} categories={CategoriesArray} />
+        <button className="See-more-button" onClick={() => event()}>
+          View all products
+        </button>
       </div>
     </div>
   );
 };
 
 function App() {
+  const [showHome, setHomeShow] = useState(true);
+  const [showProducts, setProductsShow] = useState(false);
   const { data, isLoading } = useFeaturedBanners();
   console.log(data, isLoading);
 
+  const changeToHomePage = () => {
+    setHomeShow(true);
+    setProductsShow(false);
+  };
+
+  const changeToProductsPage = () => {
+    setHomeShow(false);
+    setProductsShow(true);
+  };
+
   return (
     <div className="App">
-      <Header />
+      <Header event={changeToHomePage} />
       <div className="Body">
-        <Home />
+        {showHome && <Home event={changeToProductsPage} />}
+        {showProducts && <AllProducts products={ProductsArray} categories={CategoriesArray}/>}
```

Comment:
Then, our conditional render will be something like this:
```suggestion
        {currentPage ==='Home' && <Home event={changeToProductsPage} />}
        {currentPage ==='Products' && <AllProducts products={ProductsArray} categories={CategoriesArray}/>}
```